library/general/poddisruptionbudget/template.yaml

```diff
@@ -36,7 +36,10 @@ spec:
         violation[{"msg": msg}] {
           obj := input.review.object
           pdb := data.inventory.namespace[obj.metadata.namespace]["policy/v1"].PodDisruptionBudget[_]
-          obj.spec.selector.matchLabels == pdb.spec.selector.matchLabels
+
```

Comment:
Thanks for the PR @ctrought!
For bug fixes like this, do we need to bump the constraint template version?
> metadata.gatekeeper.sh/version: 1.0.0

+1 on adding a test to suite.yaml

Reply:
Yes, we should.

Adding versioning doc for ref: https://docs.google.com/document/d/1IYiypA-mRcdfSVfmoeyuaeG8XtA1u4GkcqH3kEkv2uw/edit#

Reply:
> Thanks for the PR @ctrought! For bug fixes like this, do we need to bump the constraint template version? @maxsmythe @sozercan @nilekhc
> 
> > metadata.gatekeeper.sh/version: 1.0.0
> 
> +1 on adding a test to suite.yaml

JFYI `suite.yaml` is mandatory since our website and AH treat that as the source of truth.

Reply:
Thanks all, will do! I was thinking it might also be good to have an extra test case to avoid over-enforcement. While not true in the current policy, if the template were to change it could be possible for all current test cases to pass while also inadvertently rejecting updates to workloads that aren't linked to any PDB. All the test cases only include tests of deployments & pdb's that are linked (e.g. Deployment violation depends on PDB, but not all Deployments have PDBs so we should include a test to ensure they are not inadvertently affected). This might be a good practice to apply to any referential based policies that are conditional. Take a look and let me know if that's alright.

Reply:
I like the idea of testing to make sure the non-enforced cases are maintained, for sure.

WRT the looser match criteria, I don't think there is not much we can do there.